src/GovUk.Education.ExploreEducationStatistics.Data.Services/MetaGuidanceSubjectService.cs

```diff
@@ -131,6 +136,16 @@ private List<LabelValue> GetVariables(Guid subjectId)
         private async Task<MetaGuidanceSubjectViewModel> BuildSubjectViewModel(ReleaseSubject releaseSubject)
         {
             var subject = releaseSubject.Subject;
+
+            var releaseFileData = await _contentDbContext
+                .ReleaseFiles
+                .Include(rf => rf.File)
+                .SingleOrDefaultAsync(rf =>
```

Comment:
I think wherever you use `SingleOrDefaultAsync` should be `SingleAsync` because the code would blow up on the next line anyway on `.Name` if no row was found.